swarm/storage/localstore.go

```diff
@@ -184,3 +184,30 @@ func (ls *LocalStore) Iterator(from uint64, to uint64, po uint8, f func(Address,
 func (ls *LocalStore) Close() {
 	ls.DbStore.Close()
 }
+
+func (ls *LocalStore) Migrate() error {
```

Comment:
motivation that this is on localstore not ldbstore is that localstore will be a different subpackage right?

Reply:
i've been thinking what the correct place for this is, and I am not sure, maybe `LDBStore` is actually better. Then again we have an exported `DbStore` in `LocalStore`, so I figured it doesn't really matter at this point.

I can move it, if necessary.

Reply:
On a second thought, for this first migration, I think we want to remove all invalid chunks => use the MRU validators and the ContentAddressed chunk validators to determine if chunks are valid or not. @nolash does that make sense?

Reply:
Yes, I think this too makes sense.

Reply:
i agree @nonsense, plus, i think we shouldn't expose direct interaction with `DbStore` outside of `LocalStore`.